docs/custom-flows/email-sms-otp.mdx

```diff
@@ -1,85 +1,67 @@
 ---
-title: Email/SMS OTP (One-time Password)
-description: Learn how to send one-time codes (OTP) to authenticate users.
+title: Build a custom email or SMS OTP authentication flow
+description: Learn how build a custom email or SMS one time code (OTP) authentication flow using using Clerk's useSignUp() and useSignIn() hooks.
 ---
 
-# Email/SMS OTP (One-time Password)
+# Build a custom email or SMS OTP authentication flow
 
-Clerk supports passwordless authentication, which lets users sign in and sign up without having to remember a password. During sign-in, users will be asked to enter their identifier (email address or phone number) to receive a one-time password (OTP), also referred to as a one-time code, and complete the authentication process.
-
-Arguably, passwordless authentication provides greater security and a better user experience than traditional passwords. However, it is not without its downsides, and often still boils down to the email providers "knowledge based factor" instead of yours.
-
-There are multiple ways to set up passwordless authentication in Clerk such as using [Clerk Components](/docs/components/overview), or by creating a [custom flow](#custom-flow) using Clerk's SDKs.
-
-The rest of this guide will explain how to set up passwordless authentication using any of the above methods. Before you start, you will need to [configure](#configuration) your instance to allow passwordless sign-ins.
-
-<Callout type="info">
-  Looking for email links? Check out our [Email links authentication](/docs/custom-flows/email-links) guide.
-</Callout>
-
-<Callout type="info">
-  Looking for two-factor authentication (2FA)? Check out our [Multi-factor authentication](/docs/custom-flows/mfa) guide.
+<Callout type="warning">
+  This guide is for users who want to build a *custom* user interface using the Clerk API. To authenticate users with one-time passwords using a *prebuilt* UI, you should use Clerk's [Account Portal pages](/docs/account-portal/overview) or [prebuilt components](/docs/components/overview).
 </Callout>
 
-## Before you start
-
-- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.
```

Comment:
Why get rid of the before you start?

Also, if we're cutting it here, we should cut it here as well:
- https://clerk.com/docs/custom-flows/sign-out
- https://clerk.com/docs/custom-flows/invitations#before-you-start
- https://clerk.com/docs/custom-flows/mfa
- https://clerk.com/docs/custom-flows/email-links

Reply:
yes, all of custom flows are being worked on one page at a time.